sphere-models/src/it/java/io/sphere/sdk/products/queries/SuggestQueryTest.java

```diff
@@ -0,0 +1,93 @@
+package io.sphere.sdk.products.queries;
+
+import io.sphere.sdk.models.LocalizedStringEntry;
+import io.sphere.sdk.products.SuggestionResult;
+import io.sphere.sdk.search.SearchKeyword;
+import io.sphere.sdk.search.SearchKeywords;
+import io.sphere.sdk.search.tokenizer.CustomSuggestTokenizer;
+import io.sphere.sdk.search.tokenizer.WhiteSpaceSuggestTokenizer;
+import io.sphere.sdk.test.IntegrationTest;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Locale;
+
+import static io.sphere.sdk.products.ProductFixtures.withSuggestProduct;
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.*;
+
+public class SuggestQueryTest extends IntegrationTest {
+    @Test
+    public void execution() {
+        withSuggestProduct(client(), product -> {
+            final SearchKeywords searchKeywords = SearchKeywords.of(
+                    Locale.ENGLISH, asList(SearchKeyword.of("Multi tool"),
+                            SearchKeyword.of("Swiss Army Knife", WhiteSpaceSuggestTokenizer.of())),
+                    Locale.GERMAN, singletonList(SearchKeyword.of("Schweizer Messer",
+                            CustomSuggestTokenizer.of(asList("schweizer messer", "offiziersmesser", "sackmesser"))))
+            );
+            assertThat(product.getMasterData().getStaged().getSearchKeywords()).isEqualTo(searchKeywords);
+
+            final SuggestQuery suggestQuery = SuggestQuery.of(LocalizedStringEntry.of(Locale.ENGLISH, "knife"))
+                    .withStaged(true);
+
+            final SuggestionResult suggestionResult = execute(suggestQuery);
+
+            assertThat(suggestionResult.getSuggestionsForLocale(Locale.ENGLISH))
+                    .matches(suggestionsList -> suggestionsList.stream()
+                            .anyMatch(suggestion -> suggestion.getText().equals("Swiss Army Knife")));
```

Comment:
No, there should not, but I tested it that way in case there are relicts of more suggestions.